content/docs/guides/contribute/contribute-code/frontend-conventions.en.md

```diff
@@ -82,17 +82,6 @@ When an endpoint call must be skipped because a variable is not defined, RTK's [
 Application translation is performed on [Weblate](https://hosted.weblate.org/projects/osrd/).
 The default language is French.
 
-## Rules and important elements
-
-#### No component should be responsible for updating the data it uses
-
-Only <u>views</u> contain the store selectors, which are then given as props to the components of the module linked to the view.
-
-#### SCSS is not scoped
-
-A `.scss` file buried in the tree structure doesn't guarantee that the classes it contains can only be accessed there, even by importing react (formally forbidden by the way: you must use SCSS import), all declared classes are accessible everywhere.
-
-Prefer a judicious choice of root class name for a given module, and use the tree structure available in the SCSS file.
 
 #### Imports must follow a specific order
```

Comment:
Deleting the "Rules and important elements" heading above makes it so this level 4 heading is now under the level 3 heading "Translation". Should we retain a level 3 heading of some kind above this one?

Oh, I now realize the next commit re-organizes the import stuff and re-introduces a level 3 heading. It would've been slightly nicer to retain the old level 3 heading in this commit and drop it in the next one - but that sounds like more trouble than it's worth now!

tl;dr feel free to ignore this comment.